utils/user_utils.jsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018-present Riff Learning, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable header/header */
+
+// Use this function to sort an array of learning group types alphabetically by the prefix
```

Comment:
Here, `lexicographically by unicode numbering` instead of `alphabetically`? It's pedantic, but alphabetically suggests to me that it might treat numbers, non-roman letters, and symbols in a peculiar way that isn't the true behavior.